### PR TITLE
adding the option --max-coverage and fixing up short option for

### PR DESCRIPTION
**Pull request by nh13**
--min-coverage

### CODE REVIEW

File: src/Parameters.cpp [hunk 1]
```diff
@@ -548,7 +550,7 @@ Parameters::Parameters(int argc, char** argv) {
     while (true) {
 
         int option_index = 0;
-        c = getopt_long(argc, argv, "hcO4ZKjH[0diN5a)Ik=wl6#uVXJY:b:G:M:x:@:A:f:t:r:s:v:n:B:p:m:q:R:Q:U:$:e:T:P:D:^:S:W:F:C:&:L:8:z:1:3:E:7:2:9:%:_:,:(:",
+        c = getopt_long(argc, argv, "hcO4ZKjH[0diN5a)Ik=wl6#uVXJY:b:G:M:x:@:A:f:t:r:s:v:n:B:p:m:q:R:Q:U:$:e:T:P:D:^:S:W:F:C:&:L:8:z:1:3:E:7:2:9:%:_:,:(:!:+:",
```
Comment (nh13) [inline src/Parameters.cpp]: see also the addition of ```!```.

**Comment by nh13** [comment]
note to self: probably should mention this is per-sample max-coverage, not overall maximum coverage.

**Comment by nh13** [comment]
@ekg could you take a look?

**Comment by nh13** [comment]
@ekg bump

**Comment by ekg** [comment]
Ah no I let this fall off. Let me try to merge manually.

**Pull request merged**